src/model/ExteriorFuelEquipment.cpp

```diff
@@ -130,8 +131,8 @@ namespace model {
       OS_ASSERT(result);
     }
 
-    bool ExteriorFuelEquipment_Impl::setFuelType(const std::string& fuelType) {
-      bool result = setString(OS_Exterior_FuelEquipmentFields::FuelUseType, fuelType);
+    bool ExteriorFuelEquipment_Impl::setFuelType(const FuelType& fuelType) {
```

Comment:
We should've been using FuelType before?

Reply:
That's my opinion, yes. Cf https://github.com/NREL/OpenStudio/pull/4940#issuecomment-1705087379